src/pages/elements/Report.js

```diff
@@ -426,7 +426,7 @@ const PostDeployJobs = ({ get_ns, post_deploy_jobs}) => {
               formatters: [headerFormat]
             },
             cell: {
-              formatters: [booleanFormat(<OnboardingStatus state={"Success"}>Success</OnboardingStatus> , <OnboardingStatus state={"Failure"}>Failure</OnboardingStatus>), cellFormat]
+              formatters: [booleanFormat(<OnboardingStatus state={"Exist"}>Exist</OnboardingStatus> , <OnboardingStatus state={"Not Exist"}>Not Exist</OnboardingStatus>), cellFormat]
```

Comment:
Using OnboardingStatus here is a bit confusing. I suggest you create a new component or use the badge directly.
I also would like to make Exist badge green and Not Found badge red.

Reply:
Will do!